src/Service/CvrHelper.php

```diff
@@ -194,14 +142,41 @@ public function collectRelevantData(array $data): array
     {
         $relevantData = [];
 
-        $relevantData['name'] = $data['GetLegalUnitResponse']['LegalUnit']['LegalUnitName']['name'];
-        $relevantData['street'] = $data['GetLegalUnitResponse']['LegalUnit']['AddressOfficial']['AddressPostalExtended']['StreetName'];
-        $relevantData['number'] = $data['GetLegalUnitResponse']['LegalUnit']['AddressOfficial']['AddressPostalExtended']['StreetBuildingIdentifier'];
-        $relevantData['floor'] = array_key_exists('FloorIdentifier', $data['GetLegalUnitResponse']['LegalUnit']['AddressOfficial']['AddressPostalExtended']) ? $data['GetLegalUnitResponse']['LegalUnit']['AddressOfficial']['AddressPostalExtended']['FloorIdentifier'] : '';
-        $relevantData['side'] = array_key_exists('SuiteIdentifier', $data['GetLegalUnitResponse']['LegalUnit']['AddressOfficial']['AddressPostalExtended']) ? $data['GetLegalUnitResponse']['LegalUnit']['AddressOfficial']['AddressPostalExtended']['SuiteIdentifier'] : '';
-        $relevantData['postalCode'] = $data['GetLegalUnitResponse']['LegalUnit']['AddressOfficial']['AddressPostalExtended']['PostCodeIdentifier'];
-        $relevantData['city'] = $data['GetLegalUnitResponse']['LegalUnit']['AddressOfficial']['AddressPostalExtended']['DistrictName'];
+        $relevantData['name'] = $data['virksomhedsnavn']['vaerdi'];
+        $relevantData['street'] = $data['beliggenhedsadresse']['CVRAdresse_vejnavn'];
+        $relevantData['number'] = array_key_exists('CVRAdresse_husnummerFra', $data['beliggenhedsadresse']) ? $data['beliggenhedsadresse']['CVRAdresse_husnummerFra'] : '';
```

Comment:
Use `$data['beliggenhedsadresse']['CVRAdresse_husnummerFra'] ?? ''` here (and in the other similar lines).

Reply:
Fixed in bf74062dec3f44a39a81dac5b0fb01ad2f0e1a5b.